### ISSUE TITLE
How to make cssom() inject at the end of the document?

### ISSUE THREAD

**Issue by cvladan** [tw-in-js/twind#476]
I have inserted

```
<style id="__twind"></style>
```

at the end of the HTML body, and that is fine.
When I use

```
dom(document.querySelector("#__twind"))
```

Twind-generated utility classes work as expected and take precedence over the other ones.

However, when I do the same with `cssom` like this
```
cssom(document.querySelector("#__twind"))
```

it reports an error. The regular `cssom()` variant inserts CSS declarations somewhere in the head, and it overrides almost everything later.

So, the question is, **how can I make `cssom()` inject at the end of the document**?
In other words, how can I give it the highest weight?

**Comment by cvladan** [comment]
I think I just realized that the project is probably abandoned.
Besides sporadic updates about sponsors, I think the author has little time for this.
It's a shame since the idea is good.

**Comment by rschristian** [comment]
> it reports an error.

What's the error message?

> how can I make cssom() inject at the end of the document?

At the end of the document? Generally this is something you want to avoid at all costs. The further down in the document it is, the higher likelihood of FOUC.

> I think I just realized that the project is probably abandoned.

Not having updates for a few months doesn't make it abandoned.

**Comment by cvladan** [comment]
So this looks good to you?

```
cssom(document.querySelector("#__twind"))
```

> "at the end of the document" is for a very good reason and obviously FOUC is not important in this use case.

**Comment by rschristian** [comment]
> So this looks good to you?

It looks like valid JS, sure, but it'd help if you could attach the error message that you're shown.
